### PR TITLE
Enable Travis-CI

### PR DESCRIPTION
**Pull request by kevmoo**
CC @eseidelGoogle

### CODE REVIEW

File: example/test_runner.dart [hunk 1]
```diff
@@ -20,49 +20,53 @@ main() {
       abbr: 'c',
       defaultsTo: 'none',
       help: 'Specify any compilation step (if needed).',
-      allowed: ['none', 'dart2js', 'dartc'],
+      allowed: [
+        'none',
+        'dart2js',
+        'dartc'
```
Comment (eseidelGoogle) [inline example/test_runner.dart]: Is this the new dartfmt behavior?  I'm surprised it doesn't require a trailing , to explode this 3-item list like this.

**Comment by eseidelGoogle** [comment]
If I were writing this change I would have split out the dartfmt application from the travisification. :/

But lgtm.  Honestly you should get someone who actually codes-fulltime to review this though.  e.g. @munificent or @lrhn

**Comment by kevmoo** [comment]
> If I were writing this change I would have split out the dartfmt application from the travisification

@eseidelGoogle It's two commits. I'll land it as two separate commits. 😄

**Comment by eseidelGoogle** [comment]
I guess my Github-fu is too weak to view it as such. :D 

Regardless, it looks fine.  Bob may have more useful commentary for you. :p

File: example/test_runner.dart (hunk 1 above)
Comment (munificent) [inline example/test_runner.dart]: dartfmt's behavior can be subtle in some cases. If I recall, there is a rule that says that if all of the trailing arguments of an argument lists are collection literals, then if any of them split (like the allowedHelp map below), then they all should. It avoids weird cases like:

```dart
function(first: [
  "stuff",
  "more",
  "stuff",
  "more",
  "stuff",
  "more",
  ], second: ["short"], third: [
  "stuff",
  "more",
  "stuff",
  "more",
  "stuff",
  "more",
])
```

**Pull request merged**